impeller/renderer/BUILD.gn

```diff
@@ -4,18 +4,84 @@
 
 import("//flutter/impeller/tools/impeller.gni")
 
-impeller_component("renderer") {
+impeller_component("allocation") {
   sources = [
     "allocator.cc",
     "allocator.h",
-    "blit_command.cc",
-    "blit_command.h",
-    "blit_pass.cc",
-    "blit_pass.h",
     "buffer.cc",
     "buffer.h",
     "buffer_view.cc",
     "buffer_view.h",
+    "device_buffer.cc",
+    "device_buffer.h",
+    "device_buffer_descriptor.cc",
+    "device_buffer_descriptor.h",
+    "formats.cc",
+    "formats.h",
+    "host_buffer.cc",
+    "host_buffer.h",
+    "platform.cc",
+    "platform.h",
+    "range.cc",
+    "range.h",
+    "texture.cc",
+    "texture.h",
+    "texture_descriptor.cc",
+    "texture_descriptor.h",
+    "vertex_buffer.cc",
+    "vertex_buffer.h",
+  ]
+
+  deps = [
+    "../base",
+    "../geometry",
+    "../image",
+    "//flutter/fml",
+  ]
+}
+
+if (impeller_enable_compute) {
+  impeller_shaders("tessellation_shaders") {
```

Comment:
compute_shaders perhaps because of the overloading if the tessellation term?

Reply:
done